diesel/src/sqlite/connection/functions.rs

```diff
@@ -66,6 +66,13 @@ where
     Ok(())
 }
 
+pub fn register_collation<F>(conn: &RawConnection, fn_name: &str, f: F) -> QueryResult<()>
```

Comment:
It is not clear for me why we need a separate function here instead of calling `RawConnectionn::register_collation_function()` directly in the implementation of `SqliteConnection`

Reply:
I was merely following the lead of `register()` and `register_aggregate()`.
Would this also apply to them? I can refactor that in this PR if desired.

Reply:
We are doing a bunch of other stuff (checking and converting formats for example) for normal functions, therefore the wrapper functions here. That is not required for registering a collation. Therefore I would only remove the `register_collation` function here.